src/bitcoin/wallet.rs

```diff
@@ -52,6 +52,8 @@ impl Wallet {
                     .create_wallet(&self.name, None, Some(true), None, None)
                     .await?;
 
+                self.bitcoind_client.rescan(&self.name).await?;
```

Comment:
📣 The `rescan` must be done **after** the seed is set. At this point in time, there is no key to check against.